.github/workflows/macosx_windows_ci.yaml

```diff
@@ -53,15 +53,12 @@ jobs:
         shell: bash
         run: |
           source activate ${ENV_NAME}
-          pip install .
+          python setup.py build_ext --inplace --force
       - name: Run Tests
         shell: bash
         run: |
-          mkdir -p empty
-          cd empty
           source activate $ENV_NAME
-          pytest --pyargs pyuvdata --cov=pyuvdata --cov-config=../.coveragerc --cov-report xml:../coverage.xml
-          cd ..
+          pytest --cov=pyuvdata --cov-config=.coveragerc --cov-report xml:coverage.xml
```

Comment:
I want to make sure that we aren't undoing something by changing this setup. Presumably it was set up this way for a good reason, it's fine if that's no longer applicable but I want to make sure that's the case. I think Matt said it was @steven-murray who set it up this way, maybe he can comment?

Reply:
It's hard to remember exactly, but I do remember having a lot of issues with running tests properly due to the structure of the repo. Since the code is not in a non-python-package `src/` directory, running `pytest` at top level would not run the _installed_ code, but rather the code directly from the repo. This can reasonably often cause problems.

I can't remember, but it may be that the `--pyargs` was a way to circumvent this, by forcing it to use the installed package rather than the repo. So I would be very careful in removing it.

Reply:
It might be that the package is not properly installing tests then with their nested directory structure?

Reply:
Ah I may have an idea. It involved the `packages` keyword in the setuptools configuration. We could change to using `find_packages()` for `find_namespace_packages()` which are both imported from setuptools. These are similar but different tools to comb the pacakge subdirectories and install files it finds. It is also possible to list paths to exclude in their calls. But if the issue is that the tests are not getting installed properly it may fix that. Anecdotally it worked on a local install. I monitored my installed packages in the `site-packages` of a conda env and verified that no tests beyond the initial `tests` folder were installed before but making this edit caused all files and tests to be installed.

Reply:
But not sure if this preserves the pytest args set in setup.cfg then :thinking:

Reply:
ah the `-c` options allows us to point to a config file.